ts/config.ts

```diff
@@ -150,30 +150,12 @@ export const totMessageFetchWorkers = pipe(
 export const shufflePinPadOnPayment =
   Config.SHUFFLE_PINPAD_ON_PAYMENT === "YES";
 
-export const privacyUrl: string = pipe(
-  Config.PRIVACY_URL,
-  t.string.decode,
-  E.getOrElse(() => "https://io.italia.it/app-content/tos_privacy.html")
-);
-
```

Comment:
There is an ESLint rules about this `privacyUrl` constant. Maybe we don't need it anymore. What do you think?

Reply:
We have this [ESLint rule](https://github.com/pagopa/io-app/blob/f2659cfd53feff5976397807916b84e234287d04/.eslintrc.js#L141). I guess we don't need it anymore. What do you think?